docs/cristin.ttl

```diff
@@ -116,13 +120,15 @@
 
 :dato_opprettet
     a               :cristinfield ;
-    rdfs:comment    "The date the resource was registered in Cristin." ;
+    rdfs:comment    "When the domain is :Result, it is the date the resource was registered in Cristin. Otherwise UNKNOWN meaning." ;
```

Comment:
Ohhh, hm, are you attempting to define the field "dato_opprettet" independent of the object in which it is used?

Hm. Hm. In general, there is no guarantee that two fields with the same name but in different tables / objects, have the same meaning.

That being said, a quick survey of fields names across the current json export shows most field names being unique.

And in particular "brukernavn_opprettet", "dato_opprettet" and "eierkode_opprettet" usually mean "who and when created this row in the table / this instance of the object". But the details may differ, like when vitenskapeligarbeid_lokal only has one "eierkode" field that applies to both "opprettet" and "siste_endring" for obvious reasons. :-| 

So, global definitions, bad gut feeling.

Reply:
I was considering creating one instance per object but the documentation becomes very verbose because we have to create weird names to distinguish between the fields. We use the domain the object that the item belongs to and we can elaborate in the comment field for each domain.